core/services/vrf/vrf_fulfillment_cost_test.go

```diff
@@ -0,0 +1,29 @@
+package vrf
+
+import (
+	"context"
+	"testing"
+
+	"github.com/ethereum/go-ethereum"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMeasureFulfillmenttGasCost(t *testing.T) {
```

Comment:
This test style is quite hard to read, would you consider chinking blocks into logical segments? I sort of see three logical groupings here: setup, execution and assertions.

Reply:
Thanks for the feedback. I factor the gas-cost test into a utility function, and broke up the new gas-cost test functions as you suggested.